cypress/e2e/mainPage/createRoute.js

```diff
@@ -0,0 +1,16 @@
+module.exports = () => {
+    it('user should be able to create a new route',() =>{
+        cy.get('#newRouteBtn').click();
+        cy.url().should('equal', 'http://localhost:3000/routes/new');
+        cy.get('#name').type('Tárkány - Párkány');
+        cy.get('#length').type('100.00');
+        cy.get('#elevation').type('100.00');
+        cy.get('#link').type('https://www.google.com/bokod-tokod');
+        cy.get('input[type=submit]').click();
+        cy.get('.navbar-brand').click();
+        cy.url().should('equal', 'http://localhost:3000/');
+        cy.get('tbody').children().last().children().eq(1).should('contain', 100);
+        cy.get('tbody').children().last().children().eq(2).should('contain', 100);
```

Comment:
Szebb lenne ha az összes érték különbözik, mert ilyenkor lehet hogy fordítva vannak, de a többi teszt szerintem lefedi mert ott különböző értékek is vannak.